app/lib/forms/possible_funding.rb

```diff
@@ -0,0 +1,15 @@
+module Forms
+  class PossibleFunding < Base
```

Comment:
Do we want a potentially very basic test for this?

Reply:
It's mostly covered by the capybara test. I can add a unit test though

Reply:
I only mentioned it because I think other pages have unit tests, and we probably would want it for code consistency. But it's not like the page is very complex...

Reply:
I've just added some, should cover everything.